docs/configuration.adoc

```diff
@@ -87,6 +87,7 @@ Options for Uptane.
 | `key_source`              | `"file"`     | Where to read the device's private key from. Options: `"file"`, `"pkcs11"`.
 | `key_type`                | `"RSA2048"`  | Type of cryptographic keys to use. Options: `"ED25519"`, `"RSA2048"`, `"RSA3072"` or `"RSA4096"`.
 | `secondary_configs_dir`   | `""`         | Directory containing individual secondary json configuration files. Example here: link:{aktualizr-github-url}/config/secondary/virtualsec.json[]
+| `secondary_config_file`   | `""`         | A path to a json file containing configuration of Secondary ECU(s) for Primary
```

Comment:
Perhaps worth pointing to the example and also explaining why we have both this and `secondary_configs_dir`. It's probably worth stating that we want to get rid of `secondary_configs_dir`, too.

Reply:
ok.